### PR TITLE
feat(css): add field-sizing property

### PR DESCRIPTION
**Pull request by chrisdavidmills**
### Description

Chrome 123 supports the `field-sizing` property, which is used to size textual form elements and textareas by their content, rather than sticking to a fixed size. This PR adds data for the new property.

- ChromeStatus: https://chromestatus.com/feature/5176596826161152
- Spec: https://drafts.csswg.org/css-ui/#field-sizing
- Explainer: https://github.com/tkent-google/explainers/blob/main/form-sizing.md

### CODE REVIEW

**Review by dipikabh — CHANGES_REQUESTED** [review]
Hi @chrisdavidmills, I have some concerns about the `appliesto` value. I'm wondering if it’s worth all the effort to perfect it. What do you think?

File: css/properties.json [hunk 1]
```diff
@@ -4605,6 +4605,22 @@
...
+  "field-sizing": {
+    "syntax": "content | fixed",
+    "media": "visual",
+    "inherited": false,
+    "animationType": "discrete",
+    "percentages": "no",
+    "groups": [
+      "CSS Basic User Interface"
+    ],
+    "initial": "normal",
```
[the remark this answers is not in the document]
Comment (chrisdavidmills) [reply css/properties.json]: I suspect that was a copy-paste error. Fixed now, thanks for spotting it.

File: css/properties.json [hunk 2]
```diff
@@ -4605,6 +4605,22 @@
...
+    "appliesto": "textFields",
```
[the remark this answers is not in the document]
Comment (chrisdavidmills) [reply css/properties.json]: Yeah, this wasn't great. I've added the suggested value `elementsWithDefaultPreferredSize` to the schema and the l10n CSS file, and used it for `field-sizing`.

[the remark this answers is not in the document]
Comment (dipikabh) [reply css/properties.json]: That's great, thanks!

**Review by dipikabh — APPROVED** [review]
Thanks for the updates and addressing the comments!

**Pull request merged**